qa/tasks/cephfs/mount.py

```diff
@@ -512,20 +512,17 @@ def run_python(self, pyscript, py_version='python3'):
         p.wait()
         return p.stdout.getvalue().strip()
 
-    def run_shell(self, args, wait=True, stdin=None, check_status=True,
-                  cwd=None, omit_sudo=True, timeout=900):
+    def run_shell(self, args, omit_sudo=True, timeout=900, **kwargs):
```

Comment:
this breaks `qa/tasks/ceph_fuse.py`.
```
2020-08-29T14:11:54.657 ERROR:teuthology.run_tasks:Saw exception from tasks.
Traceback (most recent call last):
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_master/teuthology/run_tasks.py", line 93, in run_tasks
    manager.__enter__()
  File "/usr/lib/python3.6/contextlib.py", line 81, in __enter__
    return next(self.gen)
  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2020-08-29-1945/qa/tasks/ceph_fuse.py", line 157, in task
    info["mount"].mount(mountpoint=mountpoint, mount_path=mount_path)
  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2020-08-29-1945/qa/tasks/cephfs/fuse_mount.py", line 33, in mount
    self.setup_netns()
  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2020-08-29-1945/qa/tasks/cephfs/mount.py", line 305, in setup_netns
    self._setup_brx_and_nat()
  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2020-08-29-1945/qa/tasks/cephfs/mount.py", line 136, in _setup_brx_and_nat
    """, timeout=(5*60), omit_sudo=False, cwd='/')
  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2020-08-29-1945/qa/tasks/cephfs/mount.py", line 528, in run_shell_payload
    return self.run_shell(["bash", "-c", Raw(f"'{payload}'")], **kwargs)
  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2020-08-29-1945/qa/tasks/cephfs/mount.py", line 525, in run_shell
    return self.client_remote.run(args=args, cwd=cwd, timeout=timeout, stdout=stdout, stderr=stderr, **kwargs)
TypeError: run() got multiple values for keyword argument 'cwd'
2020-08-29T14:11:54.690 ERROR:teuthology.run_tasks: Sentry event: https://sentry.ceph.com/sepia/teuthology/?q=fc4ede48879f4b36a4e727f14654335d
Traceback (most recent call last):
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_master/teuthology/run_tasks.py", line 93, in run_tasks
    manager.__enter__()
  File "/usr/lib/python3.6/contextlib.py", line 81, in __enter__
    return next(self.gen)
  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2020-08-29-1945/qa/tasks/ceph_fuse.py", line 157, in task
    info["mount"].mount(mountpoint=mountpoint, mount_path=mount_path)
  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2020-08-29-1945/qa/tasks/cephfs/fuse_mount.py", line 33, in mount
    self.setup_netns()
  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2020-08-29-1945/qa/tasks/cephfs/mount.py", line 305, in setup_netns
    self._setup_brx_and_nat()
  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2020-08-29-1945/qa/tasks/cephfs/mount.py", line 136, in _setup_brx_and_nat
    """, timeout=(5*60), omit_sudo=False, cwd='/')
  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2020-08-29-1945/qa/tasks/cephfs/mount.py", line 528, in run_shell_payload
    return self.run_shell(["bash", "-c", Raw(f"'{payload}'")], **kwargs)
  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2020-08-29-1945/qa/tasks/cephfs/mount.py", line 525, in run_shell
    return self.client_remote.run(args=args, cwd=cwd, timeout=timeout, stdout=stdout, stderr=stderr, **kwargs)
TypeError: run() got multiple values for keyword argument 'cwd'
```
/a//kchai-2020-08-29_13:54:19-rados-wip-kefu-testing-2020-08-29-1945-distro-basic-smithi/5388513/teuthology.log

i'd suggest avoid passing args using keyword argument unless it helps with readability.